routes/game/gameRoutes.js

```diff
@@ -39,7 +43,7 @@ router.put('/', async (req, res) => {
         const updGame = await GameInfo.findOne({_id: ObjectId(req.body._id)});
 
         if (updGame) {
-            const {updatedData} = req.body;
+            const { updatedData } = req.body; // TODO: Validation for this is a mess

```

Comment:
@aidan-lemay @vsoler27 Any ideas on how to validate input for a JSON object with dynamic properties?

Reply:
other than checking for properties and validating in ifs, not really. i need to think abt it a bit